packages/e2e-test-utils-playwright/src/editor/publish-post.ts

```diff
@@ -28,9 +28,10 @@ export async function publishPost( this: Editor ) {
 		'role=region[name="Editor publish"i] >> role=button[name="Publish"i]'
 	);
 
-	const urlString = await this.page.inputValue(
-		'role=textbox[name="Post address"i]'
-	);
+	const urlString = await this.page
+		.getByRole( 'region', { name: 'Editor publish' } )
+		.getByRole( 'textbox', { name: /address/i } )
```

Comment:
Nit: Could this just be `'address'`?

Reply:
Good point. I forgot that `exact` is `false` by default.